README.md

```diff
@@ -180,17 +180,17 @@ From v0.2.0, Job Manager starts to release stock docker images on [DockerHub](ht
 
 - To build the `job-manager-ui` image with `$TAG` from the root of this Github repository:
     ```
-    cd ui && docker build -t job-manager-ui:$TAG . -f Dockerfile
+    docker build -t job-manager-ui:$TAG . -f ui/Dockerfile
```

Comment:
I assume this works? I just want to check the `Dockerfile` doesn't have any relative paths that are upset because the PWD is now different...?

Reply:
Yes this works. I have to change the `PWD` because some of the commands within the Dockerfile actually refers to files in the upper lever to the Dockerfile itself.